base/common/src/main/java/com/netscape/certsrv/profile/PolicyDefault.java

```diff
@@ -153,18 +168,129 @@ public boolean equals(Object obj) {
         return true;
     }
 
+    public Element toDOM(Document document) {
+
+        Element pdElement = document.createElement("policyDefault");
+        pdElement.setAttribute("id", name);
+        pdElement.setAttribute("classId", classId);
+
+        if (text != null) {
+            Element descriptionElement = document.createElement("description");
+            descriptionElement.appendChild(document.createTextNode(text));
+            pdElement.appendChild(descriptionElement);
+        }
+
+        if (!attributes.isEmpty()) {
+            for (ProfileAttribute attribute : attributes) {
```

Comment:
The `if` statement is not really necessary since we will not enter the loop if the `attributes` is empty. Same thing with the `params` below.

Reply:
Done.